include/igl/vertex_triangle_adjacency.cpp

```diff
@@ -70,7 +70,7 @@ IGL_INLINE void igl::vertex_triangle_adjacency(
   // vfd now acts as a counter
   vfd = NI;
 
-  VF = Eigen::VectorXi(3*F.rows());
+  static_cast<DerivedVF &>(VF )= Eigen::VectorXi(3*F.rows());
```

Comment:
I'm sorry can you remind me why is this line necessary? If you need to access the derived type you can write `VF.derived() = `. I remember having issues with templated types and allocation in Eigen 3.3, not sure what's the typical way to go.